src/en/Octopii.lua

```diff
@@ -98,8 +96,38 @@ return {
 	end,
 
 	hasSearch = true,
-	isSearchIncrementing = true,
+	isSearchIncrementing = false,
 	search = function(data)
-		return parseTop(GETDocument(expandURL("/page/" .. data[PAGE] .. "/?s=" .. encode(data[QUERY]))))
+        -- Create JSON body for POST request.
+        local requestJson = "{" ..
+                                '"clicked":false,' ..
+                                '"limit":"200",' .. -- increased limit to limit to a single request
+                                '"page":0,' .. -- therefore, only request first page
+                                '"pageCount":1,' ..
+                                '"value":"' .. data[QUERY] .. '",' ..
+                                '"sort":4,' ..
+                                '"selected":{"genre":[],"status":[],"sort":[],"author":[]},' ..
+                                '"results":[],' ..
+                                '"label":"searching ...."' ..
+                            "}"
```

Comment:
```suggestion
        local requestJson = json.encode {
                                clicked = false,
                                limit = 200, -- increased limit to limit to a single request
                                page = 0, -- therefore, only request first page
                                pageCount = 1,
                                value = data[QUERY],
                                sort = 4,
                                selected = { genre={}, status={}, sort={}, author={} },
                                results = {},
                                label = "searching ...."
                            }
```
Use the json library to construct json.